pkg/standalone/standalone.go

```diff
@@ -297,7 +297,8 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		}
 		for _, container := range dockerContainerNames {
 			containerName := utils.CreateContainerName(container, dockerNetwork)
-			ok, err := confirmContainerIsRunningOrExists(containerName, true, runtimeCmd)
+			var ok bool
```

Comment:
move declaration to outside for loop or use shortcut to declare and init ':='

Reply:
did not know why I changed this. Reverted.